auth/kbase_user_federation.go

```diff
@@ -34,6 +34,36 @@ import (
 // users who have made requests to the DTS. This prevents us from having to
 // rely on a secondary data source for this information.
 
+// associates the given KBase username with the given ORCID ID
```

Comment:
Today's instance of super pedantry is noting that sometimes you write "ORCID ID" and sometimes just "ORCID". Given that these are comments, it does not matter a whit, but if you feel motivated to make a change before merging, this is one possibility.

Reply:
Thanks, @ialarmedalien . I have taken recently to just using "ORCID", cuz, you know, "ID" is already in there. I'll scan for this and clean it up--easy enough. Thanks also for reviewing all these PRs!